### PR TITLE
Sp 587 show oasys plan

### PR DESCRIPTION
**Pull request by Coupey**

### CODE REVIEW

File: app/activeplan/homepage/tab-objectives.njk [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <li class="app-task-list__item">
       <span class="app-task-list__task-name">
         <h3 class="govuk-heading-s govuk-!-margin-bottom-2">
-          {{ objective.description }}
+          {{ objective.description | safe }}
```
Comment (Coupey) [inline app/activeplan/homepage/tab-objectives.njk]: fixes a bug I saw while playing on dev

File: app/objectiveClose/index.njk [hunk 2]
```diff
@@ -32,7 +32,7 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">
         <span class="govuk-caption-l">Close objective</span>
-        {{ objective.description }}
+        {{ objective.description | safe }}
```
Comment (Coupey) [inline app/objectiveClose/index.njk]: fixes a bug I saw while playing on dev

File: app/printing/printLegacy/get.controller.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    let legacyPlan
+
+    // get the OASys sentence plan
+    try {
+      legacyPlan = await getOasysSentencePlan(id, planId, tokens)
+    } catch (error) {
+      logger.error(`Could not retrieve OASys sentence plan ${planId} for offender ${id}, error: ${error}`)
+      return res.render('app/error', { error })
+    }
```
Comment (jonhindle) [inline app/printing/printLegacy/get.controller.js]: Do we need an internal catch here?
The error is already logged by the `sentencePlanningApi`.

File: app/printing/printLegacy/get.controller.test.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      planId: 6,
+    },
+    tokens,
+  }
+  const res = {
+    render: jest.fn(),
+  }
+
+  beforeEach(() => {
+    getOasysSentencePlan.mockReset()
```
Comment (jonhindle) [inline app/printing/printLegacy/get.controller.test.js]: I think we should do `.mockReset()` calls in `afterEach` blocks

**Pull request merged**